pubsub/v2/pullstream.go

```diff
@@ -20,13 +20,16 @@ import (
 	"io"
 	"net/url"
 	"sync"
+	"sync/atomic"
 	"time"
 
 	pb "cloud.google.com/go/pubsub/v2/apiv1/pubsubpb"
 	gax "github.com/googleapis/gax-go/v2"
 	"google.golang.org/grpc"
 )
 
+const protocolVersion int64 = 1
```

Comment:
There's several references to this new protocol version but I'm not clear what it is.  Is this IBV or something else?  The comments here don't communicate it well.

Reply:
Added comment to this to clarify this isn't settable, and is used for the client library / server to negotiate which behavior to use.